lib/dal/src/func/authoring.rs

```diff
@@ -437,6 +447,12 @@ impl FuncAuthoringClient {
         let func = Func::get_by_id(ctx, func_id).await?;
         func.error_if_locked()?;
 
+        if func.is_transformation {
```

Comment:
Ah, so this is the key requirement. Attribute functions would be fine, except we want a guarantee that they will never have new arguments added.